src/einsteinpy/symbolic/einstein.py

```diff
@@ -90,3 +90,23 @@ def change_config(self, newconfig="ul", metric=None):
             new_tensor, self.syms, config=newconfig, parent_metric=metric
         )
         return new_obj
+
+    def lorentz_transform(self, transformation_matrix):
+        """
+        Performs a Lorentz transform on the tensor.
+
+        Parameters
+        ----------
+            transformation_matrix : ~sympy.tensor.array.dense_ndim_array.ImmutableDenseNDimArray or list
+                Sympy Array or multi-dimensional list containing Sympy Expressions
+
+        Returns
+        -------
+            ~einsteinpy.symbolic.tensor.BaseRelativityTensor
+                lorentz transformed tensor(or vector)
+
+        """
+        t = super(EinsteinTensor, self).lorentz_transform(transformation_matrix)
+        return EinsteinTensor(
+            t, syms=self.syms, config=self._config, parent_metric=None,
```

Comment:
`return EinsteinTensor(t.tensor(), syms=.......)`

`t` is a `BaseRelativityTensor` object while `t.tensor()` returns a sympy object.